modules/machineset-yaml-osp-sr-iov-port-security.adoc

```diff
@@ -123,3 +122,12 @@ In that case, you can apply the compute security group to the primary VM interfa
 ----
 $ openstack port set --enable-port-security --security-group <infrastructure_id>-<node_role> <main_port_ID>
 ----
+
+[NOTE]
```

Comment:
A similar note in a different module is `[IMPORTANT]`. Consider if these should be consistent.